src/models/meta_details.rs

```diff
@@ -255,6 +282,9 @@ fn selected_override_update(
                 id: video_id,
                 extra: vec![],
             }),
+            // we must set the stream_override to false after we've overridden it
+            // to make it consistent
+            guess_stream: false,
```

Comment:
I guess it's not needed to be overriden now that you check if it's none.

Reply:
yes, that's why it's `false`

Reply:
No, I meant that this line is no longer needed, as we are setting the stream_path, and this function won't trigger anymore anyway.

Reply:
You need to set a value for the field, `false` is the default + you don't want to guess the stream path when we've overridden it